client/src/components/project.js

```diff
@@ -77,7 +80,7 @@ class Project extends Component {
               <Row>
                 <div style={{alignContent: "center"}}>
 
-                    <Button type="primary" onClick={() => {parseInt(this.fundProject(this.projectId, this.state.fundValue))}}>
+                    <Button type="primary" onClick={() => {parseInt(this.fundProject(this.state.fundValue))}}>
```

Comment:
Is parseInt necessary?

Reply:
Yes, the return value is used nowhere. Let's fix this.